src/loaders/jobs/taskfarmer/task_generator.py

```diff
@@ -64,21 +64,30 @@
 VERSION_FILE = 'versions.yaml'
 COMPUTE_TOOLS_DIR = '../../compute_tools'  # relative to task_generator.py
 
-# TODO GTDB update readme to specify to get correct version of data, not just latest
-# TODO REPRODUCIBILITY need to version the databases and use the correct version with the
-#                      correct tool version
-# directory containing the unarchived GTDB-Tk reference data
-# download data following the instructions provided on
-# https://ecogenomics.github.io/GTDBTk/installing/index.html#gtdb-tk-reference-data
-GTDBTK_DATA_PATH = '/global/cfs/cdirs/kbase/collections/libraries/gtdb_tk/release207_v2'
+# volume name for the Docker containers (the internal container ref data mount directory)
+TOOL_IMG_VOLUME_NAME = '/reference_data'
 
-# DIAMOND database CheckM2 relies on
-# download data following the instructions provided on https://github.com/chklovski/CheckM2#database
-CHECKM2_DB = '/global/cfs/cdirs/kbase/collections/libraries/CheckM2_database'
+TOOLS_REQUIRING_VOLUME_MOUNT = ['checkm2', 'gtdb_tk']
```

Comment:
As above I'd get rid of this entirely and just key off the versions files

Reply:
👍